teachopencadd/talktorials/T018_automated_cadd_pipeline/talktorial.ipynb

```diff
@@ -6889,23 +6889,23 @@
-    "* `Docking`: is responsible for carrying out the docking calculations and analyzing and visualizing the results. We used the Smina program for the docking experiments, and the NGLview widget to visualize the docking poses.\n",
-    "* `InteractionAnalysis`: consists of methods to analyze and visualize the protein-ligand interactions in the docking poses generated by `Docking`. The analysis was done with the help of the PLIP package, and the interactions were visualized with the NGLview widget.\n",
-    "* `OptimizedLigands`: analyzes the whole data generated by the pipeline in order to select the best analogs based on criteria specified in the input settings.  \n",
+    "* **`Specs`**: reads and processes the input data for further use in the pipeline, and creates output paths to store all the data generated throughout the whole process.\n",
+    "* **`Protein`**: uses the input protein data to create a `Protein` object that provides extended information (in form of attributes) on the protein, as well as various methods to manipulate, analyze and visualize the protein data.\n",
+    "* **`Ligand`**: creates a `Ligand` object from the input ligand data, with extended attributes and methods similar to the `Protein` object.\n",
+    "* **`BindingSiteDetection`**: provides methods to automatically perform all the necessary processes for the selection of the best protein binding site according to the input specifications. For our demonstrations in this talktorial, we used the *DoGSiteScorer* functionality of the *ProteinsPlus* webserver as our binding site detection method.\n",
+    "* **`LigandSimilaritySearch`**: contains all required functionalities to automatically execute a ligand similarity search to find suitable analogs according to the input specifications. In this talktorial, the similarity search was done using the *PubChem* web-services via the provided API backend. This class is also responsible for filtering the analogs based on several calculated drug-likeness scores.\n",
+    "* **`Docking`**: is responsible for carrying out the docking calculations and analyzing and visualizing the results. We used the *Smina* program for the docking experiments, and the *NGLview* widget to visualize the docking poses.\n",
+    "* **`InteractionAnalysis`**: consists of methods to analyze and visualize the protein&mdash;ligand interactions in the docking poses generated by `Docking`. The analysis was done with the help of the *PLIP* package, and the interactions were visualized with the *NGLview* widget.\n",
+    "* **`OptimizedLigands`**: analyzes the whole data generated by the pipeline in order to select the best analogs based on criteria specified in the input settings.  \n",
```

Comment:
Hi @Armin-Ariamajd, any reason you changed that back to bold instead of code? I thought we are referring here to the classes, ergo code, aren't we?

Reply:
Hi @dominiquesydow,

yes we are; they are still in code style, I just made them bold as well. There were a lot of other code style text in that block as well, so I thought making the title of each bulletpoint bold makes it easier on the eyes. If you think it's not appropriate then I'll change them back.